windows/window_manager_plugin.cpp

```diff
@@ -176,6 +186,14 @@ std::optional<LRESULT> WindowManagerPlugin::HandleWindowProc(HWND hWnd,
       _EmitEvent("focus");
     } else {
       _EmitEvent("blur");
+
+      /// logic to hide window on outside click(HideOnDeactivate)
+      if (window_manager->is_hide_on_deactivate_) {

```

Comment:
Let's also listen to this `blur` event on buzz_flutter_buzzer, so we can handle the logic from there directly.

Reply:
Moved logic to our desktop app.